Xamarin.Forms.Core/BorderElement.cs

```diff
@@ -1,14 +1,86 @@
-﻿namespace Xamarin.Forms
+﻿using System;
+
+namespace Xamarin.Forms
 {
 	static class BorderElement
 	{
+		const int DefaultBorderRadius = 5;
+		const int DefaultCornerRadius = -1;
+
 		public static readonly BindableProperty BorderColorProperty =
 			BindableProperty.Create("BorderColor", typeof(Color), typeof(IBorderElement), Color.Default,
 									propertyChanged: OnBorderColorPropertyChanged);
 
+		public static readonly BindableProperty BorderWidthProperty = BindableProperty.Create("BorderWidth", typeof(double), typeof(IBorderElement), -1d);
+
+		[Obsolete("BorderRadiusProperty is obsolete as of 2.5.0. Please use CornerRadius instead.")]
+		public static readonly BindableProperty BorderRadiusProperty = BindableProperty.Create("BorderRadius", typeof(int), typeof(IBorderElement), defaultValue: DefaultBorderRadius,
```

Comment:
Obsoleted properties can stay where they were originally defined. it feels odd to add new deprecated api (even if internal)

Reply:
or is it required to move it there so we can do our smart stuff to be backward compatible ?